content/html/concepts/tags/terms/basefont/basefont.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '<basefont>'
+Description: 'Used to be used to set the font of text. This is now deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+  - 'Typography'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<basefont>`** tag was used to set the default font characteristics for the text of an HTML document. Since it is now deprecated, [CSS](https://www.codecademy.com/resources/docs/css)  properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead. To set the font of specific parts of the webpage the [`<font>`](https://www.codecademy.com/resources/docs/html/tags/font) would be used in combination with the `<basefont>` tag.
+
+
+## Syntax
+
+The tag must be a child of the [head](https://www.codecademy.com/resources/docs/html/tags/head) tag. Any combination of the following three attributes could be used in the `<font>` tag, with at least one being required:
+
+- `color`: What [color](https://www.codecademy.com/resources/docs/css/colors) the font/text should be. The color can be set to a named [color], hexadecimal or RGB.
+- `face`: What font the text should be. For example, to have the text be `verdana`, the `face` attribute would be set to `face="verdana"`.
+- `size`: Sets the size of the font/text. It can use absolute units ranging from 1-7. Absolute units are used by `size="5"` or any other number 1-7.

```

Comment:
```suggestion
- `color`: Sets the [color](https://www.codecademy.com/resources/docs/css/colors) for the font/text using a named color, hexadecimal, or RGB.
- `face`: Sets the font for the text (e.g., `face="verdana"`).
- `size`: Sets the size of the font/text using absolute units ranging from 1-7 (e.g., `size="5"`).
```

A few points with this section:

- Remember to use oxford commas when listing things inline with `and` or `or` (e.g., `... named color, hexadecimal, or RGB.`).
- List items should assume a similar format/sentence structure.
- Instead of dedicating a whole new sentence to an example in a list item, I suggest using the latin abbreviation `e.g.,` in parentheses, instead.